spec/models/vm_or_template_spec.rb

```diff
@@ -479,8 +479,8 @@
       expect(FactoryGirl.create(:vm_vmware, :miq_group => group1, :tenant => tenant2).tenant).to eq(tenant1)
     end
 
-    it "assigns no tenant by default" do
-      expect(FactoryGirl.create(:vm_vmware).tenant).to be_nil
+    it "uses default tenant via tenancy_mixin" do
+      expect(FactoryGirl.create(:vm_vmware).tenant).to eq(Tenant.root_tenant)
```

Comment:
@gtanzillo this fixes the other test failure on travis.   We explicitly changed the behavior of this test by changing the way "no assigned tenants" becomes the root tenant.